{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,4 +1,4 @@
-# Title: "{{cookiecutter.project_name}}"
+# {{cookiecutter.project_name}}: {{cookiecutter.project_description}}
```

Comment:
the project description is typically a bit to long to go in a title. I think ti'd be best to put this on a newline below the title.

Reply:
Can! I just checked the other HydroMT plugins so we could also change the suggestion to a short title addition (HydroMT plugin for {pluginname})